deploy/examples/create-external-cluster-resources.py

```diff
@@ -1593,6 +1611,14 @@ def _gen_output_map(self):
         self.out_map["RBD_METADATA_EC_POOL_NAME"] = (
             self.validate_rbd_metadata_ec_pool_name()
         )
+        self.out_map["TOPOLOGY_POOLS"] = self._arg_parser.topology_pools
```

Comment:
csi `"domainSegments": [{` it itself an array do we need to support that?

Reply:
@travisn @Madhu-1 more thoughts,
Do we need to add more than one fd for a pool?

Reply:
In a cluster, I would normally expect them to use a single failure domain. If an admin wants to set up some advanced scenario with multiple failure domains in the same cluster, I would expect that they would have separate storage classes, where one failure domain is used by each storage class. I don't know a scenario where the same storage class would need to define topology with multiple failure domains mapped to the pools.

Reply:
yes i agree with @travisn

Reply:
Okay thanks for the feedback, I will continue with the current design